.github/workflows/test_bwc.yml

```diff
@@ -0,0 +1,45 @@
+name: BWC
+on:
+  push:
+    branches:
+      - "**"
+  pull_request:
+    branches:
+      - "**"
+
+jobs:
+  Build-ff-linux:
+    strategy:
+      matrix:
+        java: [11,17,21]
+      fail-fast: false
+
+    name: Test Flow Framework BWC
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Setup Java ${{ matrix.java }}
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: ${{ matrix.java }}
+
+      - name: Checkout Flow Framework
+        uses: actions/checkout@v4
+
+      - name: Assemble Flow Framework
+        run: |
+          plugin_version=`./gradlew properties -q | grep "opensearch_build:" | awk '{print $2}'`
+          echo plugin_version $plugin_version
```

Comment:
```suggestion
          echo "Plugin Version:" $plugin_version
```

Reply:
I like it matching the variable name as-is.  Here's how it looks in the debug log output:
```
Run plugin_version=`./gradlew properties -q | grep "opensearch_build:" | awk '{print $2}'`
plugin_version 3.0.0.0-SNAPSHOT
```